operations/tables.md

```diff
@@ -0,0 +1,52 @@
+<!-- AUTOMATICALLY GENERATED, DO NOT MODIFY -->
+# Tables
+
+## Get tables
+
+This operation returns a list of tables.
```

Comment:
I wonder if we might add the word "restaurant" just to make it crystal clear we are talking about restaurant tables.
```suggestion
This operation returns a list of restaurant tables.
```